test/verify/check-login

```diff
@@ -364,6 +365,52 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
                                     "couldn't parse login input: Malformed input",
                                     "couldn't parse login input: Authentication failed")
 
+    @skipImage("No SELinux", "debian-stable", "debian-testing", "ubuntu-stable", "ubuntu-1804", "ubuntu-2004")
+    @skipImage("No semanage", "fedora-coreos")
+    @unittest.expectedFailure
```

Comment:
This should be accompanied by a comment leading to a bugzilla. As above, it's not quite obvious how to reproduce this outside of cockpit -- sudo as sysadm_u does work in general.

Reply:
Fixed with more explicit code